docs/reference/msbuild-targets.md

```diff
@@ -55,9 +55,10 @@ Note that the `Owners` and `Summary` properties from `.nuspec` are not supported
 | RequireLicenseAcceptance | PackageRequireLicenseAcceptance | false | |
 | license | PackageLicenseExpression | empty | Corresponds to `<license type="expression">` |
 | license | PackageLicenseFile | empty | Corresponds to `<license type="file">`. You may need to explicitly pack the referenced license file. |
-| LicenseUrl | PackageLicenseUrl | empty | `licenseUrl` is being deprecated, use the PackageLicenseExpression or PackageLicenseFile property |
+| LicenseUrl | PackageLicenseUrl | empty | `PackageLicenseUrl` is being deprecated, use the PackageLicenseExpression or PackageLicenseFile property |
 | ProjectUrl | PackageProjectUrl | empty | |
-| IconUrl | PackageIconUrl | empty | |
+| icon | PackageIcon | empty | You may need to explicitly pack the referenced icon image file.|
```

Comment:
nit: space between the . at the end and |
Note that this won't affect the markdown formatting, so if it doesn't bug you, don't fix it, lol :D

Reply:
Either all rows in the table should have a trialing period, or none should. Since this column it's a full sentence, and not just a single word like the other columns, I think we should have periods, but I think it's more important to be consistent.

Reply:
good feedback. will handle this in a separate PR.